synapse/lib/view.py

```diff
@@ -1254,6 +1254,89 @@ async def _calcChildViews(self):
 
                 todo.append(child)
 
+    async def insertParentFork(self, useriden, name=None):
```

Comment:
Should this have the same restriction as fork() in that the parent cannot have any open merge requests?

Reply:
yeah probably should, added a check